x-pack/test/api_integration/apis/uptime/rest/monitor_states_generated.ts

```diff
@@ -53,18 +53,6 @@ export default function ({ getService }: FtrProviderContext) {
         nonSummaryIp = checks[0][0].monitor.ip;
       });
 
-      it('should return all IPs', async () => {
-        const filters = makeApiParams(testMonitorId);
-        const url = getBaseUrl(dateRangeStart, dateRangeEnd) + `&filters=${filters}`;
-        const apiResponse = await supertest.get(url);
-        const res = apiResponse.body;
-
-        const uniqueIps = new Set<string>();
-        res.summaries[0].state.checks.forEach((c: any) => uniqueIps.add(c.monitor.ip));
-
-        expect(uniqueIps.size).to.eql(4);
-      });
-
```

Comment:
not valid anymore?

Reply:
Yeah, we don't actually show individual IPs on the overview page anymore, so now we just return the latest summary per geographic location. To show the IPs we'd actually need to do an extra query.